lib/fxpay/adapter.js

```diff
@@ -0,0 +1,166 @@
+(function() {
+  'use strict';
+
+  var exports = window.fxpay.utils.namespace('fxpay.adapter');
+
+  var API = require('fxpay/api').API;
+  var products = require('fxpay/products');
+  var receipts = require('fxpay/receipts');
+  var settings = require('fxpay/settings');
+  var utils = require('fxpay/utils');
+
+
+  function FxInappAdapter() {
+    //
+    // Adapter for Firefox Marketplace in-app products.
```

Comment:
Some of this comment seems confusing to me.This is the in-app adapter isn't it - as opposed to the one that hasn't been written for the marketplace specifics? The comment seems to mention the latter which muddies things for me - though granted I'm easily confused :)